lib/install/config/webpacker.yml

```diff
@@ -0,0 +1,41 @@
+# Note: You must restart bin/webpack-dev-server for changes to take effect
+
+default: &default
+  dev_server_host: localhost
+  dev_server_port: 8080
+  dev_server_https: false
+
+  source_path: app/javascript
+  source_entry_path: packs
+  public_output_path: packs
+
+  extensions:
+    - .coffee
+    - .erb
+    - .js
+    - .jsx
+    - .ts
+    - .vue
+    - .sass
+    - .scss
+    - .css
+    - .png
+    - .svg
+    - .gif
+    - .jpeg
+    - .jpg
+
+development:
+  <<: *default
+
+test:
+  <<: *default
+
+  public_output_path: packs-test
+
+production:
+  <<: *default
+
+  dev_server_host: 0.0.0.0
+  dev_server_port: 8080
+  dev_server_https: true
```

Comment:
My vote is to remove these alternate dev server settings from production. People can add them if they're using the dev server in production (for some reason). Also, if `0.0.0.0` works more broadly, let's use it in the default settings instead of `localhost`.